ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/events/CreateTableEvent.java

```diff
@@ -62,11 +67,26 @@ private List<HivePrivilegeObject> getInputHObjs() {
     List<HivePrivilegeObject> ret   = new ArrayList<>();
     PreCreateTableEvent       event = (PreCreateTableEvent) preEventContext;
     Table                     table = event.getTable();
-    String                    uri   = getSdLocation(table.getSd());
+    Database               database = event.getDatabase();
+    String                      uri = getSdLocation(table.getSd());
```

Comment:
I think there was some unconventional spacing, but can we keep the same thing to maintain redability
```
    List<HivePrivilegeObject> ret   = new ArrayList<>();
    PreCreateTableEvent       event = (PreCreateTableEvent) preEventContext;
    Table                     table = event.getTable();
    Database                  database = event.getDatabase();
    String                    uri = getSdLocation(table.getSd());
```